### PR TITLE
Fix dataset dtype reflecting when viewing assets from .dataset API

### PR DESCRIPTION
**Pull request by IonesioJunior**
## Description
The error happened due to the fact that dtype was a string `< class ...Tensor >`. The `<>` were mixed with HTML syntax, creating an error while trying to render it in jupyter notebook.

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: packages/syft/src/syft/core/node/common/util.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def upload_result_to_s3(
         asset_name=asset_name,
         dataset_name=dataset_name,
         node_id=domain_id,
-        dtype=data_dtype,
```
Comment (gmuraru) [inline packages/syft/src/syft/core/node/common/util.py:118]: Q: Could we delete the `data_type`? (it is used anywhere else?)

File: packages/syft/src/syft/core/node/common/util.py [hunk 2]
```diff
@@ -262,7 +262,7 @@ def upload_to_s3_using_presigned(
             asset_name=asset_name,
             dataset_name=dataset_name,
             node_id=client.id,
-            dtype=data_dtype,
```
Comment (gmuraru) [inline packages/syft/src/syft/core/node/common/util.py:265]: Also here

File: packages/syft/src/syft/core/node/common/util.py (hunk 1 above)
Comment (IonesioJunior) [reply packages/syft/src/syft/core/node/common/util.py:118]: Good catch! Yup, we must delete it.

Comment (IonesioJunior) [reply packages/syft/src/syft/core/node/common/util.py:118]: Solved by: [54a6c1e](https://github.com/OpenMined/PySyft/pull/6997/commits/54a6c1e92d38de1bc27952bf332c902ec0a53a67)

File: packages/syft/src/syft/core/node/common/util.py (hunk 2 above)
Comment (IonesioJunior) [reply packages/syft/src/syft/core/node/common/util.py:265]: Solved by: [54a6c1e](https://github.com/OpenMined/PySyft/pull/6997/commits/54a6c1e92d38de1bc27952bf332c902ec0a53a67)

**Comment by rasswanth-s** [comment]
@IonesioJunior ,I think when module name is too large , it overlaps with ID , when we view the datasets, 
Maybe we could print just the class Name without the path or reformat the pandas table?
Screenshot 2022-11-08 at 4 51 58 PM

**Comment by IonesioJunior** [comment]
Ohh, yeah! Thank you for noticing it @rasswanth-s! ❤️

**Comment by IonesioJunior** [comment]
Looks better now, but maybe in the future we can change this table to change its size dynamically according to the size of the output
datasets_vis

**Review by rasswanth-s — APPROVED** [review]
Well done @IonesioJunior 🤟

**Pull request merged**